src/viewport-observer.js

```diff
@@ -76,7 +76,7 @@ export function observeWithSharedInOb(element, viewportCallback) {
     );
   }
 
-  const win = element.ownerDocument.defaultView;
+  const win = devAssert(element.ownerDocument.defaultView);
```

Comment:
There's actually `toWin` function just for this case.

Reply:
awesome. updated 👍